src/config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // URL to submit reasoning requests to.
-  JPHYLOREF_SUBMISSION_URL: 'https://phyloref.rc.ufl.edu/hooks/reason',
+  JPHYLOREF_SUBMISSION_URL: 'http://phyloref.ggvaidya.com/reason',
```

Comment:
To avoid having to change the code whenever the reasoner is migrated to a new server, it would be much preferable to use a subdomain which forwards or is CNAME'd to the actual target server. I've set up reasoner.phyloref.org to be a CNAME for phyloref.ggvaidya.com, but you will probably have to create a virtual host record on the webserver there before it will respond properly.

Reply:
Yes, that makes perfect sense! I've made the change in 61b35ee85.

I can see how this would benefit from setting up a Lets Encrypt cert for the reasoner, but I'm a little nervous about how this would work on UF HiperGator, which has an HTTPS proxy server that proxies to HTTP for us. So I'm going to leave this on HTTP for now, but we can look into it again once our resolver is back up on UF HiperGator. I've filed it as phyloref/curation-tool#183.